components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -42,26 +51,26 @@ export class LayoutFacadeService {
           notifications &&  notifications.some(n => n.type === Type.license);
         this.updateContentHeight(
           this.showLicenseNotification ? Height.Banner : Height.Navigation);
-      });
+    });
   }
 
   hasGlobalNotifications(): boolean {
     return this.headerHeight === Height.Banner;
   }
 
-  ShowPageLoading(showLoading: boolean): void {
-    this.store.dispatch(new ShowPageLoading(showLoading));
+  ShowPageLoading(showLoading: boolean) {
```

Comment:
I'm an angular newbie-- can you explain why we drop the `void` for some, but not all of these? It looks like it clarifies whether we expect anything returned.

Reply:
(or is the thing that looks like the arg the return?)

Reply:
@blakestier The void represents the type of value returned, in this case nothing.